### ISSUE TITLE
Translation errors in QuickFolders 5.4 (Settings screen)

### ISSUE THREAD

**Issue by RealRaven2000** [RealRaven2000/QuickFolders#123]
In the French version a user reported seeing the following error when they tried to open QuickFolders settings:

```
Erreur d’analyse XML : erreur dans le traitement d’une référence d’entité externe
Emplacement : chrome://quickfolders/content/options.xhtml
Numéro de ligne 17, Colonne 72 :
<!DOCTYPE prefwindow SYSTEM "chrome://quickfolders/locale/options.dtd" >
```

**Comment by RealRaven2000** [comment]
QuickFix for french locale:

[QuickFolders-wx-5.4.1pre1.zip](https://github.com/RealRaven2000/QuickFolders/files/6170876/QuickFolders-wx-5.4.1pre1.zip)

To install this version, download the zip file. Then drag the file into Thunderbird Add-ons Manager.

**Comment by RealRaven2000** [comment]
Just released fixed version 5.4.1 on ATN
